config_db.go

```diff
@@ -97,6 +97,7 @@ var (
 	sleepScheduleDisabledKey           = bsonutil.MustHaveTag(ServiceFlags{}, "SleepScheduleDisabled")
 	sleepScheduleBetaTestDisabledKey   = bsonutil.MustHaveTag(ServiceFlags{}, "SleepScheduleBetaTestDisabled")
 	systemFailedTaskRestartDisabledKey = bsonutil.MustHaveTag(ServiceFlags{}, "SystemFailedTaskRestartDisabled")
+	degradedModeDisabledKey            = bsonutil.MustHaveTag(ServiceFlags{}, "DegradedModeDisabled")
```

Comment:
Can we call this something more specific like `cpuDegradedModeDisabled` or something? Basically all the service flags are service degradation flags.